Sources/Plasma/FeatureLib/pfConsole/pfConsoleCommands.cpp

```diff
@@ -278,19 +278,17 @@ PF_CONSOLE_FILE_DUMMY(Main)
 // Name can be an alias specified by saying $foo
 //
 plKey FindObjectByName(const ST::string& name, int type, const ST::string& ageName,
-                       const char** statusStr, bool subString = false)
+                       ST::string& statusStr, bool subString = false)
 {
     if (name.empty())
     {
-        if (statusStr)
-            *statusStr = "Object name is nil";
+        statusStr = "Object name is nil";
```

Comment:
These would be a good place to use `ST_LITERAL` or the `_st` literal suffix

Reply:
Ah yep. I used it more consistently elsewhere, but got lazy when I saw the 6000-line file 🙃

Reply:
Okay, added `ST_LITERAL` in the relevant places. I didn't adjust every single `PrintString` call though - that would be quite a bit of work and not really worth it. (This is the dev console - I don't think we need to micro-optimize performance and memory usage that much.)